.github/workflows/codeql-analysis.yml

```diff
@@ -24,17 +24,17 @@ jobs:
         language: [ 'python' ]
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
+      - name: Checkout repository
```

Comment:
don't commit unrelated changes

Reply:
Maybe we should have a fix for these type of lint error soon. Will remove this for now.